OfficeDev/TeamsFx
# chore: improve cli preview telemetry

- unify handling of start/end events for preview stages (use localTelemetryReporter so `duration` is automatically added)
- align telemetry with vsc
    - separate PrerequisiteValidationError for account check and node check
    - `preview` event will contain `preview-last-event-name` and `preview-check-results`
    - add `preview-dev-cert-status` for cert event
    - remove `PreviewPrerequisitesCheckTime` because every end events contains `duration` now
 
Test:
![image](https://user-images.githubusercontent.com/9698542/175264807-2e7e2b22-b8d0-4de6-be75-d50342b0cb6b.png)

"teamsfx-cli/preview" event
![image](https://user-images.githubusercontent.com/9698542/175265284-e8e1771b-1ab9-4c09-9a90-5cb79416a8ab.png)
